src/components/NewCardForm.js

```diff
@@ -1,11 +1,53 @@
-import './NewCardForm.css';
+import { useState } from "react";
+import PropTypes from 'prop-types';
+
+import "./NewCardForm.css";
+
+const NewCardForm = ({ boardId, createNewBoard }) => {
+  const [message, setMessage] = useState("");
+  const [invalidForm, setInvalidForm] = useState(true);
+
+  const generateMessage = (e) => {
+    setMessage(e.target.value)
+    if (e.target.value.length > 0 && e.target.value.length <= 40) {
+      setInvalidForm(false);
+    } else {
+      setInvalidForm(true);
+    }
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(message);
+    const data = {
+      message: message,
+      board_id: boardId,
+      likes_count: 0,
+    };
+    createNewBoard(data);
```

Comment:
Should there be something here for setting the board list display? I'm not sure, but it seems like handleSubmit won't trigger a re-render of the boardlist to display the newly created board, right?